### PR TITLE
mh_copyright: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/mh_copyright.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# mh_copyright
+
+> Adjust copyright headers for MATLAB or Octave code.
+> More information: <https://misshit.org>.
+
+- Update the year (range) to include the current year for a specific file:
+
+`mh_copyright --primary-entity="{{entity}}" --update-year {{path/to/file.m}}`
```
Comment (EmilySeville7cfg) [inline pages/common/mh_copyright.md:8]:
```suggestion
`mh_copyright --primary-entity="{{entity}}" --update-year {{path/to/file_or_directory1.m path/to/file_or_director2.m ...}}`
```

File: pages/common/mh_copyright.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Update the year (range) to include the current year for all files:
```
Comment (EmilySeville7cfg) [inline pages/common/mh_copyright.md:10]: It is not clear where they are updated. Maybe in the current directory?

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/mh_copyright.md (hunk 2 above)
Comment (navarroaxel) [reply pages/common/mh_copyright.md:10]: in-place

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)

**Pull request merged**